co-authors-plus.php

```diff
@@ -314,221 +319,30 @@ public function is_post_type_enabled( $post_type = null ) {
 	}
 
 	/**
+	 * DEPRECATED - Use cap_remove_authors_box() instead!
```

Comment:
For these deprecated functions, can we call `_deprecated_function()` to throw a warning and add a `@deprecated` and/or `@see` tag (Kinda like https://github.com/WordPress/WordPress/blob/505c2ddb61b1509c49bd0d9e8328c6a1856f6827/wp-admin/includes/deprecated.php#L97)

Reply:
Sure - will update that! What version should I use in the deprecated notice? 

`_deprecated_function( __FUNCTION__, '??????', 'cap_remove_authors_box()' );`

Reply:
Lets use 3.3.0 as the version as I don't think we've nailed down where this will be integrated to the code can be updated again when it is merged if needed to align to the latest version number.